### PR TITLE
feat: 🦀 Persistent billable metric

### PR DESCRIPTION
**Pull request by MorganeLecurieux**
https://github.com/getlago/lago/issues/77
Screenshot 2022-09-01 at 12 03 17

https://www.notion.so/getlago/Spec-Persistent-billable-metric-to-increment-or-decrement-e41b1d974b5e4a4c8c6b4920b3b80eff

### CODE REVIEW

File: src/components/wallets/TopupWalletDialog.tsx [hunk 1]
```diff
@@ -19,9 +19,7 @@ import { intlFormatNumber } from '~/core/intlFormatNumber'
 gql`
   mutation createCustomerWalletTransaction($input: CreateCustomerWalletTransactionInput!) {
     createCustomerWalletTransaction(input: $input) {
-      collection {
-        id
-      }
+      id
```
Comment (ansmonjol) [inline src/components/wallets/TopupWalletDialog.tsx:22]: Hum is that change related to this PR ? Also I feel it's wrong as I remember changing it to a collection last week 🤔

Comment (MorganeLecurieux) [reply src/components/wallets/TopupWalletDialog.tsx:22]: yep I think the BE branch was not up to date then, checking with Vincent

**Review by ansmonjol — APPROVED** [review]
